examples/fungible-token/Cargo.toml

```diff
@@ -5,8 +5,13 @@ authors = ["Near Inc <hello@nearprotocol.com>"]
 edition = "2018"
 
 [dev-dependencies]
+anyhow = "1.0"
+near-primitives = "0.5.0"
```

Comment:
yeah an extension of what Phuong mentioned, this shouldn't be included
```suggestion
```

Reply:
As @ChaoticTempest mentioned in the other thread I actually need this to pull `FinalExecutionStatus`

Reply:
This seems like something that shouldn't be exposed through workspaces

Reply:
Hmm, do you mean that workspaces should have its own type for expressing the call execution status?

CC @ChaoticTempest

Reply:
yeah, this is not an ideal pattern for workspace users to have to import a nearcore internal crate. Not a big deal, but something that should be changed at some point

Reply:
ooof, this was a pattern I wasn't expecting to cover with `asserts!` with `FinalExecutionStatus`. Think we can defer this later until a new minor version of workspaces is released and we can convert these `asserts!` over to using `is_success` calls

Reply:
> Think we can defer this later until a new minor version of workspaces is released and we can convert these `asserts!` over to using `is_success` calls

Yeah, we can defer for sure and `is_success` will help in this case, but `CallExecutionDetails::status` (which is a part of the public API) will still be a type from `near-primitives` so I think eventually we would want to replace it with something else.